README.md

```diff
@@ -200,6 +200,32 @@ For publishing data using a producer which is defined for the stream router conf
 
 `(send :default "test-topic" 1 "key" "value")`
 
+## Tracing
+[Open Tracing](https://opentracing.io/docs/overview/) enables to identify the amount of time spent in various stages of the work flow.
+
+Currently, the execution of the handler function can be traced. If the message consumed has the corresponding tracing headers, then the E2E life time of the message from the time of production till the time of consumption can be traced.
+
+Tracing has been added to the following flows:
+
+1. Normal basic consume
+2. Retry via rabbitmq
+3. Produce to rabbitmq channel
+4. Produce to another kafka topic
+
+By default, tracing is done via [Jaeger](https://www.jaegertracing.io/). The corresponding `:default-tracer-config` has to be set to publish the traces to the desired Jaeger cluster.
+To enable custom tracer, a tracer provider function name can be set in `:tracer-provider`. The corresponding function will be executed in runtime to create a tracer. In the event of any errors while executing the custom tracer provider, a Noop tracer will be created.
+
+To enable tracing, the following config needs to be added to the `config.edn` under `:ziggurat` key.
+
+```clojure
```

Comment:
this is not clojure format, edn format :p

Reply:
the configuration below also has the same format. thats why used it. should i change it to :edn?